pydantic/_internal/_typing_extra.py

```diff
@@ -58,7 +40,7 @@ def is_any(tp: Any, /) -> bool:
     #> True
     ```
     """
-    return _is_typing_name(tp, name='Any')
+    return tp is typing.Any or tp is t_e.Any
```

Comment:
I wonder if it's worth importing everything by name and avoiding the attribute access, in my test it reduced the time from about 30ns to 20ns?